superset/db_engine_specs/postgres.py

```diff
@@ -93,10 +93,15 @@ class PostgresEngineSpec(PostgresBaseEngineSpec):
 
     column_type_mappings = (
         (
-            re.compile(r"^double precision", re.IGNORECASE),
+            re.compile(r"^float", re.IGNORECASE),
             DOUBLE_PRECISION(),
             GenericDataType.NUMERIC,
         ),
+        (
+            re.compile(r"^double precision", re.IGNORECASE),
+            FLOAT(),
+            GenericDataType.NUMERIC,
+        ),
```

Comment:
Is there a reason why `float` is mapped to `DOUBLE_PRECISION()`, and `double precision` is mapped to `FLOAT()`?

Reply:
ha good catch. I'll fix that. thanks.

Reply:
I have the git hooks enabled now. However if I make a change to this file the pre-commit hooks fail.

For instance I checkout master, added a blank line to the file superset/db_engine_specs/postgres.py 

when I commit I get this error.

 git commit -m 'test'
seed isort known_third_party.............................................Passed
isort....................................................................Passed
mypy.....................................................................Failed
- hook id: mypy
- exit code: 1

superset/commands/exceptions.py:52: error: Call to untyped function "normalized_messages" in typed context
superset/commands/importers/v1/utils.py:70: error: No overload variant of "__getitem__" of "list" matches argument type "str"
superset/commands/importers/v1/utils.py:70: note: Possible overload variants:
superset/commands/importers/v1/utils.py:70: note:     def __getitem__(self, int) -> str
superset/commands/importers/v1/utils.py:70: note:     def __getitem__(self, slice) -> List[str]
superset/commands/importers/v1/utils.py:70: note:     def __getitem__(self, int) -> Any
superset/commands/importers/v1/utils.py:70: note:     def __getitem__(self, slice) -> List[Any]
superset/annotation_layers/annotations/commands/create.py:75: error: Argument 1 to "append" of "list" has incompatible type "Type[AnnotationDatesValidationError]"; expected "ValidationError"
superset/charts/schemas.py:450: error: Incompatible types in assignment (expression has type "marshmallow.fields.List", base class "Schema" defined the type as "Set[Union[Any, str]]")
superset/charts/schemas.py:575: error: Call to untyped function "Tuple" in typed context
superset/charts/api.py:569: error: Call to untyped function "normalized_messages" in typed context
Found 6 errors in 5 files (checked 1 source file)

pip-compile-multi verify.............................(no files to check)Skipped
Check docstring is first.................................................Passed
Check for added large files..............................................Passed
Check Yaml...........................................(no files to check)Skipped
Debug Statements (Python)................................................Passed
Fix End of Files.........................................................Passed
Trim Trailing Whitespace.................................................Passed
black....................................................................Failed
- hook id: black
- files were modified by this hook

reformatted superset/db_engine_specs/postgres.py
All done! ✨ 🍰 ✨
1 file reformatted.

Reply:
Hmm, this does look strange. I'm not sure what the problem is. What version of Python are you on? And are you on pip or conda?